### PR TITLE
simple: fast fixes

### PR DESCRIPTION
**Pull request by colleenXu**
should be merged quickly, since these are simple + don't change behavior much + fix a few potential bugs

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -211,83 +227,79 @@ export const APIMETA: MetaDataItemsObject = {
...
       RHEA: ['xrefs.rhea'],
-      KEGG: ['xrefs.kegg_reaction'],
+      'KEGG.REACTION': ['xrefs.kegg_reaction'],
       REACT: ['xrefs.reactome'],
       name: ['name'],
     },
   },
   BiologicalProcess: {
-    id_ranks: ['GO', 'MetaCyc', 'REACT', 'KEGG', 'name'],
+    id_ranks: ['GO', 'MetaCyc', 'REACT', 'name'],
```
Comment (andrewsu) [inline src/config.ts:244]: can you clarify why KEGG should be deleted from this list of BiologicalProcess identifiers (and MetaCyc) from the CellularComponent identifiers below?

Comment (colleenXu) [reply src/config.ts:244]: I reviewed the corresponding API's /metadata/fields endpoints and couldn't find those ID prefixes. 
- BiologicalProcess's possible [fields](https://biothings.ncats.io/go_bp/metadata/fields)
- CellularComponent's possible [fields](https://biothings.ncats.io/go_cc/metadata/fields)
- AnatomicalEntities's possible [fields](https://biothings.ncats.io/uberon/metadata/fields)
- Cell's possible [fields](https://biothings.ncats.io/cell_ontology/metadata/fields)

Comment (colleenXu) [reply src/config.ts:244]: Reviewing those API's /metadata/fields endpoints is also how some of the other pull requests #68 , #66 , #69 came to be...

Comment (newgene) [reply src/config.ts:244]: @colleenXu `xrefs.kegg_pathway` is listed under https://biothings.ncats.io/go_bp/metadata/fields

```
"xrefs.kegg_pathway": {
"index": true,
"type": "keyword"
},
```

Comment (colleenXu) [reply src/config.ts:244]: That's true but KEGG pathways don't sound like they should be biological procseses. Pathways are actually a semantic class under molecular activity instead...and we have a pathway semantic type that uses KEGG pathway IDs...

Comment (andrewsu) [reply src/config.ts:244]: I think there is an argument to say that KEGG Pathways could be considered biological processes (and `KEGG.MODULE` appears under biological process in biolink model https://github.com/biolink/biolink-model/blob/master/biolink-model.yaml#L7559). But I think this is an issue that we shouldn't spend too much time debating unless we need to. Colleen, is there a motivation to fix something by removing these IDs, or just general cleanup/housekeeping?  If there's nothing pressing driving this issue, let's remove these changes from the PR and leave only edits that truly are "fast fixes" that are clear wins.

Comment (colleenXu) [reply src/config.ts:244]: After reviewing this:

- it's general housekeeping / preventing potential bugs or inefficiencies from having BTE try to do ID resolution that never works
- I think I was mistaken on Pathway's info (it's a biological [process](https://github.com/biolink/biolink-model/blob/615629a4e3ad198b68c3ef08c4e8935816c7deb3/biolink-model.yaml#L7414)). I agree that KEGG (which I assume is KEGG pathway) could stay as a biological process ID. I will adjust this branch now...

Comment (colleenXu) [reply src/config.ts]: @newgene @andrewsu with the recent commit, KEGG has been added back under BiologicalProcess.

Does that resolve this issue?

**Comment by newgene** [comment]
@colleenXu For other IDs removed:

>remove IDs
>  BiologicalProcess KEGG
>  **CellularComponent MetaCyc**
>  **AnatomicalEntity NCIT**
>  **Cell EFO**

If the reason to remove `MetaCyc`, `NCIT`, `EFO` is the same as `KEGG` ("cannot find matching field in the corresponding API"), I would not remove them as well. It's likely the underlying API has an issue to load the corresponding fields, then we should fix the data instead, or find another API.

But if their existence breaks some queries, it's strong enough reason to remove it (or better "comment out") for now.

**Comment by colleenXu** [comment]
@newgene I have changed the branch, so this will merge to biolink_v2.1 for testing. 

I will add those removed IDs back in, with comments that I didn't find those fields in the corresponding APIs

**Pull request merged**